### DISCUSSION TITLE
Extension homepage

### DISCUSSION THREAD

**Discussion by faraplay** [OctoNezd/oldlander#50] [General]
Why is the homepage for the extension (on Firefox) https://github.com/OctoNezd/old-reddit-mobile instead of https://github.com/OctoNezd/oldlander ?

**Comment by OctoNezd** [comment]
Was initially more-or-less proof-of-concept in private repo, then I decided to name it properly and I still hadnt updated AMO

**Comment by OctoNezd** [comment]
fixed
